src/core/string_set.cc

```diff
@@ -133,4 +133,18 @@ void StringSet::ObjDelete(void* obj, bool has_ttl) const {
   sdsfree((sds)obj);
 }
 
+void* StringSet::ObjectClone(const void* obj, bool has_ttl) const {
```

Comment:
It looks like this method doesn't use any member of StringSet, maybe we don't need it as virtual and make it static or even move out from the StringSet?

Reply:
this is the design of the DenseSet, where for each derived class we provide virtual functions that know how to handle objects. DenseSet is base class for ScoreMap, StringMap and StringSet and each one provide functions that help dense set handle its objects.